### DISCUSSION TITLE
Make a runnable application from the cnossos branch

### DISCUSSION THREAD

**Discussion by tomasanda** [Universite-Gustave-Eiffel/NoiseModelling#328] [Q&A]
Hi,

is there please any documentation how to make a runnable application from the current cnossos branch?

Thank you
Tom

**Comment by pierromond** [comment]
Yes !  https://github.com/Nitnelav/NoiseModelling-installer from @Nitnelav

**Comment by Nitnelav** [comment]
Yep, but note that this will only create an executable from the last release, it will not include updates that were made since then (you won't find https://github.com/Ifsttar/NoiseModelling/pull/326 there for example)

**Reply by tomasanda** [reply]
Ok, it is clear now. Thank you guys.

**Comment by nicolas-f** [comment]
This discussion is about writing a guide to update de NoiseModelling geoserver package ?

If so

1- Copy wps scripts to /data_dir/scripts/wps
2- Recompile NoiseModelling then copy all jar files to webapps/geoserver/WEB-INF/lib

Its a short description, maybe a more elaborate guide would be usefull ?

**Reply by tomasanda** [reply]
Thank you @nicolas-f this short description is enough. Frankly, the main reason why I asked was that I wanted to use the new WPS script #326. But then I figured it out as you can see in this [Dockerfile](https://github.com/tomasanda/docker-noisemodelling/blob/develop/Dockerfile). I copied the **Import_OSM_Pbf.groovy** script to the wps scripts folder and it worked like a charm.
